lib/utilities/healthcheckHandler.js

```diff
@@ -121,7 +121,8 @@ function healthcheckHandler(clientIP, req, res, log, statsClient) {
         });
     }
 
-    const deep = (req.url === '/ready');
+    // CLDSRV-740: S3C with the healthcheck on s3 port needs the internal route
+    const deep = (req.url === '/ready') || (_config.healthcheckS3Internal && req.url === '/_/healthcheck/deep');
```

Comment:
I don't think this can actually happen: `healthchandler` is called only from `routeAdminRequest`, depending on the url:

```
        switch (req.url) {
            case '/live':
            case '/ready':
                healthcheckHandler(clientInfo.clientIP, req, res, log, statsClient);
                break;
```

Reply:
ok got this, it is on next chunk, with 2 different ways to call the function.
In that context:
- Either (and preferably) we can keep the same route on both cases, update s3frontend to use the /live and /ready path, and avoid the new config option
- Or we should make `deep` a parameter of the healtcheckHandler function, set appropriately (and differently) by both `internalHandlers` and `routeAdminRequest`, with no need for an option